README.md

```diff
@@ -1 +1,199 @@
-# react-native-sdk-button
\ No newline at end of file
+
+# Yoti Button SDK for React Native
+
+Yoti button component for React Native [Android]([https://github.com/getyoti/android-sdk-button](https://github.com/getyoti/android-sdk-button)) and [iOS]([https://github.com/getyoti/ios-sdk-button](https://github.com/getyoti/ios-sdk-button)).
+
+![Yoti Logo](https://www.yoti.com/wp-content/themes/yoti/assets/images/logo.svg)
+
+The purpose of the mobile SDK is to provide 3rd party applications the ability to request attributes from a Yoti user through Yoti mobile App. It is an interaction between a 3rd Party app and the Yoti app facilitated by a very lightweight SDK. This repository contains the module which enables your users to share their identity details with your application in a secure and trusted way.
+
+## Prerequisite
+
+Before you begin, kindly make sure you have at the minimum created an application and scenario according to the [documentation](https://www.yoti.com/developers).
```

Comment:
I think it would be better to link to the Hub part of the docs, now that we've split up the docs into many products. 
```suggestion
Before you begin, kindly make sure you have at the minimum created an application and scenario on the Yoti Hub according to the [documentation](https://developers.yoti.com/yoti/getting-started-hub).
```

Reply:
Done 👍